src/gtk2.mk

```diff
@@ -27,6 +27,7 @@ define $(PKG)_BUILD
         --disable-test-print-backend \
         --disable-gtk-doc \
         --disable-man \
+		--disable-visibility \
```

Comment:
minor — tabs here, spaces in adjacent lines

Reply:
Whoops, didn't realize that
Will fix it before merging